frontend/src/app/constants/constants.d.ts

```diff
@@ -6,7 +6,7 @@ type Race =
   | "white"
   | "unknown"
   | "refused";
-type Ethnicity = "hispanic" | "not_hispanic";
+type Ethnicity = "hispanic" | "not_hispanic" | "refused";
```

Comment:
Is there something else we should use instead of "refused" for "prefers not to answer"?

Reply:
The reason I started using `"refused"` for these fields is we had been using `"Refused to answer"`, which mapped to `"refused"` on the backend, for Race options. It's not necessarily the right call, but importantly it's being sent over to Report Stream as "unknown" regardless of what string we use here.

Reply:
Thanks for the context